README.md

```diff
@@ -17,7 +17,9 @@ Users are able to use this App as a Todo-List or a Note Taking application.
 
 ## Build, Testing, & Development
 
-1. Need to have [Nodejs](https://nodejs.org/en/), [NPM](https://www.npmjs.com/), [Grunt](http://gruntjs.com/), and [Bower](https://bower.io/) installed on your system globally. This guide assumes you are running a 16.04 LTS 64bit Ubuntu system. If running a Debian/Ubuntu system and you do not have the packages installed globally, run these commands below:
+1. Install Git on the system by running the command: sudo apt install git
```

Comment:
Yes, it needs to be clear that the user must have git installed. Thank you for catching my attention on this. 

I would prefer to remove this line and just simply add the command `sudo apt install git` right after the command `sudo apt update`.